packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -358,16 +358,24 @@ export function SelectBase<T, Rest = {}>({
   );
 }
 
-function defaultFormatCreateLabel(input: string) {
-  return (
-    <div style={{ display: 'flex', gap: '8px', alignItems: 'center' }}>
-      <div>{input}</div>
-      <div style={{ flexGrow: 1 }} />
-      <div className="muted small" style={{ display: 'flex', gap: '8px', alignItems: 'center' }}>
-        Hit enter to add
+function defaultFormatCreateLabel(createOptionPosition: 'first' | 'last') {
+  let hint;
+  if (createOptionPosition === 'first') {
+    hint = 'Hit enter to add';
+  } else {
+    hint = 'Click to add';
+  }
```

Comment:
This is a great change. It doesn't even need to be dependant on `createOptionPosition` - the message should just be changed to "Select to add"